libdnf/transaction/db/comps_group_package.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright (C) 2017-2020 Red Hat, Inc.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+
+#include <algorithm>
+#include <memory>
+
+#include "comps_group_package.hpp"
+
+#include "libdnf/transaction/transaction.hpp"
```

Comment:
Please could you reorder includes?

Reply:
clang-format keeps it this way actually.
If the include file name is the same as the current file (minus the suffix), it's placed on top.